chef/cookbooks/crowbar/templates/default/systemd.service.erb

```diff
@@ -0,0 +1,25 @@
+[Unit]
```

Comment:
Damn, this file should replace `chef/cookbooks/crowbar/files/default/crowbar.service`, or at least should get merged with it related to the lock files.